MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+### Customizing ItemsSource of SfComboBox
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the [IDataFormSourceProvider]`Link`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string Country { get; set; }
+    
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+        public override object GetSource(string sourceName)
+        {
+            var list = new List<string>();
+            if (sourceName == "ItemName")
+            {
+                list.Add("India");
+                list.Add("USA");
+                list.Add("Japan");
+            }
+            return list;
+        }
+    }
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("ItemName", DataFormEditorType.ComboBox);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using GenerateDataFormItem event
+
+You can also set the ItemsSource for combo box editor by using the [GenerateDataFormItem]`Link` event in the `DataFormComboBoxItem`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            var list = new List<string>();
+            list.Add("Home");
+            list.Add("Food");
+            list.Add("Utilities");
+            list.Add("Education");
+            comboBoxItem.ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Loading complex type property values in combo box editor
+
+You can display the complex type property values in combo box editor by using the [GetSource]`Link` method of IDataFormSourceProvider class, which is used to get source list as complex property values for combo box editor and set it to `IDataFormSourceProvider` property of SfDataForm. You need to use `GenerateDataFormItem `event to set [DisplayMemberPath]`Link` and [SelectedValuePath]`Link` property value of DataFormComboBoxItem for complex type property.
```

Comment:
You can display the complex type property values in combo box editor by using the [GetSource]`Link` method of IDataFormSourceProvider class, which is used to get source list as complex property values for combo box editor and set it to `IDataFormSourceProvider` property of SfDataForm. 

Change to

You can display the complex type property values in combo box editor by using the DisplayMemberPath and SelectedValuePath properties of DataFormComboBoxItem

Reply:
definition changed